Cereal/CerealDecoder.swift

```diff
@@ -42,14 +42,14 @@ public struct CerealDecoder {
     private let reverseRange: StrideThrough<Int>
     private let indexMap: [String: Int]?
 
-    private func itemForKey(key: String) -> CoderTreeValue? {
+    fileprivate func itemForKey(key: String) -> CoderTreeValue? {
```

Comment:
But it's a `fileprivate` method :) Also, `NSCoder` has no itemForKey method, afaik.